MultiTarget/NoWarn.props

```diff
@@ -0,0 +1,14 @@
+<Project>  
+  <!-- Disable warnings for 'This call site is reachable on all platforms.' caused by underlying platform. -->
+  <PropertyGroup Condition="'$(IsUwp)' == 'true' AND $([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'windows'">
+    <NoWarn>$(NoWarn);CA1416;</NoWarn>
```

Comment:
Want to suppress `NETSDK1219` too? It warns that UWP .NET 9 is in preview.

Reply:
```suggestion
    <NoWarn>$(NoWarn);CA1416;NETSDK1219;</NoWarn>
```